CDP4Dal.Tests/Permission/PermissionServiceTestFixture.cs

```diff
@@ -318,6 +327,80 @@ public void VerifyModifyIfOwner()
             Assert.IsTrue(this.permissionService.CanRead(this.elementDef));
         }
 
+        [Test]
+        public void VerifyModifyIfOwnerForRequirement()
+        {
+            this.session.Setup(x => x.ActivePersonParticipants).Returns(new List<Participant> { this.participant });
+            Assert.IsFalse(this.permissionService.CanWrite(this.model));
+            Assert.IsFalse(this.permissionService.CanRead(this.model));
+
+            var permission =
+                this.participantRole.ParticipantPermission.Single(x => x.ObjectClass == ClassKind.Requirement);
+            var specPermission =
+                this.participantRole.ParticipantPermission.Single(x => x.ObjectClass == ClassKind.RequirementsSpecification);
+
+            permission.AccessRight = ParticipantAccessRightKind.MODIFY_IF_OWNER;
+            specPermission.AccessRight = ParticipantAccessRightKind.MODIFY;
+
+            //Requirement has no owner
+            Assert.IsFalse(this.permissionService.CanWrite(this.requirement));
```

Comment:
as per 10-25, an OwnedThing has an owner of 1..1, so it can never be null. Either we throw an exception (something like IncompleteModelException, or we return false)